examples/ds3231/ds3231.ino

```diff
@@ -81,4 +82,4 @@ void loop () {
 
     Serial.println();
     delay(3000);
-}
+}
```

Comment:
For consistency, I would avoid removing the terminating newline character. Also note that all the files here adhere to the Unix end-of-line convention, which is to use LF as a line _terminator_ (not a line _separator_): every proper text file should end with LF.

Reply:
Good catch, that was unintentional and I’ll fix it.

Reply:
@edgar-bonet PR updated!